app/helpers/host_helper.rb

```diff
@@ -1,4 +1,11 @@
 module HostHelper
   include_concern 'TextualSummary'
   include_concern 'ComplianceSummaryHelper'
+
+  def get_host_info
+    devices = get_devices
+    os_info = get_os_info
+    vmm_info = get_vmm_info
+    [devices, os_info, vmm_info]
```

Comment:
I'd say that we don't need the temp variables here.

Also I'd say that the methods don't need the `get_` prefixes.